archetypal/trnsys.py

```diff
@@ -135,36 +136,23 @@ def convert_idf_to_trnbuild(
             "kwargs": dict(
                 Variable_Name="Zone Thermostat Heating Setpoint Temperature",
                 Reporting_Frequency="hourly",
-                save=True,
             ),
         },
         {
             "ep_object": "Output:Variable".upper(),
             "kwargs": dict(
                 Variable_Name="Zone Thermostat Cooling Setpoint Temperature",
                 Reporting_Frequency="hourly",
-                save=True,
             ),
         },
     ]
-    _, idf = run_eplus(
-        idf_file,
-        weather_file,
-        output_directory=None,
-        ep_version=ep_version,
-        output_report=None,
-        prep_outputs=outputs,
-        design_day=False,
-        annual=True,
-        expandobjects=True,
-        return_idf=True,
-    )
+    idf = IDF(idf_file, epw=weather_file, ep_version=ep_version, prep_outputs=outputs)
 
     # Check if cache exists
     # idf = _load_idf_file_and_clean_names(idf_file, log_clear_names)
     # Outpout reports
     htm = idf.htm
-    sql = idf.sql
+    sql = idf.sql  # todo: Is this needed?
```

Comment:
@louisleroy5 There are a couple of examples like this where variables are defined but they are not used (greyed out).

Reply:
It is true, we don't need this line.
I'll check trnsys.py and delete the unused code.